src/v3/src/components/PasswordEnrollment/PasswordEnrollment.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box, Button, FormHelperText } from '@mui/material';
+import { PasswordInput } from '@okta/odyssey-react-mui';
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+
+import { useOnSubmit, useValue } from '../../hooks';
+import { useTranslation } from '../../lib/okta-i18n';
+import {
+  ChangeEvent,
+  PasswordEnrollmentElement,
+  SubmitEvent,
+  UISchemaElementComponent,
+} from '../../types';
+import InputPassword from '../InputPassword';
+
+const PasswordEnrollment: UISchemaElementComponent<{
```

Comment:
maybe name it as `PasswordWithConfirmation`? I feel it can be reused for reset password

Reply:
Absolutely! And yep! It does get reused for reset password as well.